docs/site/Server.md

```diff
@@ -58,6 +58,37 @@ export class HelloWorldApp extends RestApplication {
 
 ## Configuration
 
+### Enable HTTPS
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the crendential details.
+
+In the following app we configure HTTPS for a bare miminum app using a key +
+certificate chain variant.
+
+```ts
+import {RestApplication, RestServer, RestBindings} from '@loopback/rest';
+import * as fs from 'fs';
+
+export async function main() {
+  const options = {
+    rest: {
+      protocol: 'https',
+      key: fs.readFileSync('./key.pem'),
```

Comment:
Does the `.pem` file need to be read as `utf-8` or is that just `.cert` files?

Reply:
It can be either - https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options.